src/Resources/views/base.html.twig

```diff
@@ -163,7 +163,7 @@
                 </div>
 
                 {% block footer %}
-                    <footer class="whatwedo-utility-paragraph text-center mb-2">
+                    <footer class="whatwedo-utility-paragraph !text-neutral-700 text-right mb-2 mx-4">
```

Comment:
Why does it require the `!important` flag here? Should that text colour overwrite be on the link instead too (with `text-current`) the `!important` flag being removed?

`!important` flags are considered harmful, as difficult to impossible to overwrite.

Reply:
@renestalder I actually think this is not needed. Don't know why this was added in the demo sites 🤔 
I'll remove it 👍